### PR TITLE
add "*principal*" to list of sanitized field names

### LINKED ISSUE #1641: [META 689] Ignore *principal* headers by default
See meta issue for the description and details:
- Meta issue: https://github.com/elastic/apm/issues/689

### PR DESCRIPTION
**Pull request by beniwohli**
## Related issues

closes #1641

### CODE REVIEW

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -870,12 +870,13 @@ WARNING: We recommend always including the default set of validators if you cust
...
+                                                                  "\*key",
                                                                   "*token*",
-                                                                  "*session*",
-                                                                  "*credit*",
-                                                                  "*card*",
-                                                                  "*auth*",
+                                                                  "\*session*",
+                                                                  "\*credit*",
+                                                                  "\*card*",
+                                                                  "\*auth*",
```
Comment (basepi) [inline docs/configuration.asciidoc:878]: Why this change? I assumed asciidoc must be consuming the `*` or something but it looks fine for me.

**Review by basepi — APPROVED** [review]

File: docs/configuration.asciidoc (hunk 1 above)
Comment (beniwohli) [reply docs/configuration.asciidoc:878]: yes, I realized in the preview that `*something*` is asciidoc for **bold**. Masking one of the `*` fixes that. For some reason, masking the `*` for `token` was not necessary...

Comment (beniwohli) [reply docs/configuration.asciidoc:878]: Ugh. Turns out the asciidoc preview in PyCharm renders things differently than our docs pipeline. I'll revert this change.
